### DISCUSSION TITLE
Export my DBeaver settings for use on another computer.

### DISCUSSION THREAD

**Discussion by florentfougeres** [dbeaver/dbeaver#35236] [Q&A] [answered]
Hello, 

I've set the colors and font of the SQL editor on one computer. I'd like to use these same settings on another computer. Is it possible to export and re-import them? 

Thank you in advance for your help

**Comment by TB-HunBoka** [comment, marked as answer]
Hello @florentfgrs 
I exported manually the whole workspace to do that, because there are some settings not in the current built in export method. Like: colors, data formats and possible more.
You can find those files in the dbeaver workspace .metadata\.plugins\org.eclipse.core.runtime\ I think.
https://dbeaver.com/docs/dbeaver/Configuration-files-in-DBeaver/
https://github.com/dbeaver/dbeaver/wiki/Workspace-Location
